exchanges/bitfinex/bitfinex_test.go

```diff
@@ -1109,13 +1146,6 @@ func TestWsCancelOffer(t *testing.T) {
 	}
 }
 
-func TestUpdateTradablePairs(t *testing.T) {
```

Comment:
My main issue with this is that if I wanted to test this function, I must first look at `TestMain` and see that its tested there, and then run any other test - makes it a bit weird

Reply:
I also think it'd be good to revert this even though technically it's less efficient for the sake of more straight forwarding testing. `TestUpdateTicker` is the only test which requires `UpdateTradablePairs` so it can be added there and removed from `TestMain`